VideoLocker/src/main/java/org/edx/mobile/discussion/DiscussionTextUtils.java

```diff
@@ -137,8 +135,8 @@ public void updateDrawState(TextPaint ds) {
                         context.getResources(), finalStringRes, valuesMap));
                 // If time is not available, then reduce the whitespaces
                 // surrounding it's placeholder in the template to one.
-                if (TextUtils.isEmpty(formattedText) || TextUtils.isEmpty(authorLabel)) {
-                    formattedText = removeDuplicateWhitespace(formattedText);
+                if (TextUtils.isEmpty(formattedTime)) {
+                    formattedText = removeDuplicateWhitespaces(formattedText);
```

Comment:
Btw we can fully localize the iOS implementation by localizing the `whitespace` character.
That way, we will get rid of the following assignment in the code:
```swift
let blankSpace = NSAttributedString(string : " ")
```